prow/cmd/branchprotector/protect.go

```diff
@@ -252,7 +252,7 @@ func (p *protector) protect() {
 
 // UpdateOrg updates all repos in the org with the specified defaults
 func (p *protector) UpdateOrg(orgName string, org config.Org) error {
-	if org.Policy.Unmanaged != nil && *org.Policy.Unmanaged {
+	if org.Policy.Unmanaged != nil && *org.Policy.Unmanaged && !org.HasManagedBranches() {
```

Comment:
I think this also needs to check if there are managed repos, as you can set this on repo level:
```suggestion
	if org.Policy.Unmanaged != nil && *org.Policy.Unmanaged && !org.HasManagedRepos() && !org.HasManagedBranches() {
```

Reply:
Updated.